src/ucp/tag/rndv.c

```diff
@@ -157,16 +161,16 @@ ucs_status_t ucp_proto_progress_rndv_get(uct_pending_req_t *self)
 
     offset = rndv_req->send.state.offset;
 
-    if ((offset == 0) && ((uintptr_t)rndv_req->send.buffer % UCP_ALIGN) &&
+    if ((offset == 0) && ((uintptr_t)rndv_req->send.buffer % align) &&
         (rndv_req->send.length > UCP_MTU_SIZE )) {
-        length = UCP_MTU_SIZE - ((uintptr_t)rndv_req->send.buffer % UCP_ALIGN);
+        length = UCP_MTU_SIZE - ((uintptr_t)rndv_req->send.buffer % align);
```

Comment:
can you pls check if in your case, the buffer from the user is indeed misaligned?